CHANGES.md

```diff
@@ -30,6 +30,9 @@ Selected user-facing changes:
 
 ### Enhancements
 
+[#3303](https://github.com/cylc/cylc-flow/pull/3259) - improve CLI
```

Comment:
It seems you missed out the change on the PR ID from a copy 'n paste:

```suggestion
[#3303](https://github.com/cylc/cylc-flow/pull/3303) - improve CLI
```

Reply:
Oops, good spotting!